samples/apply-style-to-wms-layer/src/main/java/com/esri/arcgismaps/sample/applystyletowmslayer/components/ApplyStyleToWMSLayerViewModel.kt

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.applystyletowmslayer.components
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.viewModelScope
+import com.arcgismaps.mapping.ArcGISMap
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.layers.WmsLayer
+import com.arcgismaps.mapping.layers.WmsSublayer
+import com.arcgismaps.toolkit.geoviewcompose.MapViewProxy
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialogViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.launch
+
+class ApplyStyleToWmsLayerViewModel(app: Application) : AndroidViewModel(app) {
+    // WMS layer displaying Minnesota's county boundaries with multiple styles.
+    private val wmsLayer = WmsLayer(
+        url = "https://imageserver.gisdata.mn.gov/cgi-bin/mncomp?SERVICE=WMS&VERSION=1.3.0&REQUEST=GetCapabilities",
+        layerNames = listOf("mncomp")
+    )
+
+    // Map used by the MapView.
+    val arcGISMap = ArcGISMap(BasemapStyle.ArcGISLightGray).apply {
+        // apply a min scale
+        minScale = 7_000_000.0
+        // Add the WMS layer to the map's operational layers
+        operationalLayers.add(wmsLayer)
+    }
+
+    // MapViewProxy to perform view operations (like setting the viewpoint) safely from a ViewModel.
+    val mapViewProxy = MapViewProxy()
+
+    // Message dialog ViewModel for error reporting.
+    val messageDialogVM = MessageDialogViewModel()
+
+    // Flow of available WMS styles.
+    private val _stylesFlow = MutableStateFlow<List<String>>(emptyList())
```

Comment:
Since the other flow is not suffixed with "Flow" we should be consistent:
```suggestion
    private val _styles = MutableStateFlow<List<String>>(emptyList())
```